managed/cmd/pmm-managed-starlark/main.go

```diff
@@ -118,20 +117,13 @@ func runChecks(l *logrus.Entry, data *checks.StarlarkScriptData) ([]check.Result
 		return nil, errors.Wrap(err, "error initializing starlark env")
 	}
 
-	res := make([][]map[string]interface{}, len(data.QueriesResults))
-	for i, queryResult := range data.QueriesResults {
-		if res[i], err = agentpb.UnmarshalActionQueryResult(queryResult); err != nil {
-			return nil, err
-		}
-	}
```

Comment:
All queries return different result structure, it's become hard to parse and properly unmarshal them here. So I moved unmarhaling to managed, right after we getting action result.